openid-federation-1_0.xml

```diff
@@ -2541,7 +2541,22 @@
                 <spanx style="verb">superset_of</spanx> with identical array
                 values.
               </t>
-
+	      <t>
+		Some JSON libraries may have issues comparing JSON objects.
+		For this reason, support for applying metadata policy to
+		metadata values that are JSON objects is mandatory only for the
+		<spanx style="verb">value</spanx>,
```

Comment:
merging `value` operators also requires comparison. Since they can (only) be merged when their values are equal.